packages/code/src/Code.spec.tsx

```diff
@@ -1,32 +1,79 @@
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, fireEvent } from '@testing-library/react';
 import { typeIs } from '@leafygreen-ui/lib';
 import Code from './Code';
 
+// I had to add this in to prevent another error being thrown on the click event (in addition to the TypeError)
+window.prompt = jest.fn();
```

Comment:
I defined a mock function for window.prompt because the error I saw was telling me that it wasn't defined and I wanted to see what would happen. Adding in that definition got rid of the error, but I wasn't sure why, so I left it in and decided to come back and ask for y'alls thoughts once I had the PR submitted. Here's the exact error I get when I comment out that line: 
```
      Error: Not implemented: window.prompt
          at module.exports (/Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:9:17)
          at /Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/browser/Window.js:594:7
          at copy (/Users/dominoweir/leafygreen-ui/node_modules/react-copy-to-clipboard/node_modules/copy-to-clipboard/index.js:93:14)
          at /Users/dominoweir/leafygreen-ui/node_modules/react-copy-to-clipboard/lib/Component.js:69:52
          at HTMLUnknownElement.callCallback (/Users/dominoweir/leafygreen-ui/node_modules/react-dom/cjs/react-dom.development.js:336:14)
          at invokeEventListeners (/Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:193:27)
          at HTMLUnknownElementImpl._dispatch (/Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:119:9)
          at HTMLUnknownElementImpl.dispatchEvent (/Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:82:17)
          at HTMLUnknownElementImpl.dispatchEvent (/Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/living/nodes/HTMLElement-impl.js:30:27)
          at HTMLUnknownElement.dispatchEvent (/Users/dominoweir/leafygreen-ui/node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:157:21) undefined
```

Reply:
Seems like it's not provided by [jsdom](https://github.com/jsdom/jsdom). I think this is fine. 

I might suggest we put it into beforeEach instead so it's reset before every test run, and then on afterEach we clear it out so it's put back to the state we found it in.